### PR TITLE
Added the first classes

### PR DESCRIPTION
**Pull request by jfoyarzo**
For this milestone of the project, the following changes were introduced:
- Created super class `Person` with the required instance variables and methods
- Created sub-classes `Student` and `Teacher` that inherits from the `Person` superclass
- The subclass `Teacher` has the method `can_use_services?` overridden to return always `true`

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
```
Comment (mikethreels) [inline person.rb]: - [ ] you are calling the method `is_of_age?` however the actual method you should be calling is called `of_age?` instead.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', _parent_permission: true)
+    super(age, name)
```
Comment (mikethreels) [inline student.rb]: - [ ] parent_permission should be included as an argument when calling super as well.

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', _parent_permission: true)
+    super(age, name)
+    @specialization = specialization
```
Comment (mikethreels) [inline teacher.rb]: - [ ] parent_permission should be included as an argument when calling super as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Felipe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @jfoyarzo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**